build_tooling/vcpkg_caching.sh

```diff
@@ -6,7 +6,7 @@ pushd $VCPKG_ROOT
 PLATFORM_VCPKG_ROOT=`cygpath -wa . 2>/dev/null || pwd`
 
 [[ -x vcpkg ]] || ./bootstrap-vcpkg.sh -disableMetrics
-nuget="`which mono 2>/dev/null` `./vcpkg fetch nuget | tail -n 1`"  # which mono will return empty on windows
+nuget="`which mono 2>/dev/null` `./vcpkg fetch nuget | tail -n 1`" # which mono will return empty on windows
```

Comment:
?

Reply:
I had to debug this line as it was breaking on the ec2 runners.
I guess that when cleaning up, I deleted a space